src/ParallelDataCollector.cpp

```diff
@@ -475,14 +475,25 @@ namespace splash
         DCParallelGroup group;
         if (dataName)
         {
+            /** create `obj_name` as a group if not exists
+             *
+             * if the `obj_name` with the given `group_path` not exists than
```

Comment:
well, maybe it is fine as long as we also document that behaviour in the public API of write[Global]Attribute